recipes/mono.recipe

```diff
@@ -30,19 +30,11 @@ class Recipe(recipe.Recipe):
         'lib/pkgconfig/mono.pc',
     ]
 
-    patches = [
-        'mono/0001-System.ServiceModel-Fix-DuplexClient-initialization-.patch',
-        'mono/0002-System.ServiceMode-Fix-XML-parsing-of-EndpointAddres.patch',
-        'mono/0003-System.ServiceModel.Discovery-Fix-deadlock-receiving.patch',
-        'mono/0004-System.ServiceModel.Discovery-Fix-exception-when-Loc.patch',
-        'mono/0005-System.ServiceModel.Discovery-Add-integration-test-f.patch',
-        'mono/0006-System.ServiceMode-Create-unit-test-for-46971.patch',
-        'mono/0007-System.ServiceModel.Discovery-Fix-messages-validatio.patch',
-    ]
-
     def prepare(self):
         if self.config.target_platform == self.config.platform and self.config.target_platform == Platform.WINDOWS:
             raise InvalidRecipeError()
+        if self.config.target_platform == self.config.platform and self.config.target_platform == Platform.DARWIN:
+            raise InvalidRecipeError()
```

Comment:
since mono-osx recipe have the same name it will use this recipe instead of mono.recipe is the same procedure as windows